components/menu/show-hide-behavior.md

```diff
@@ -0,0 +1,134 @@
+---
+title: Show and Hide Behavior
+page_title: Menu - Show and Hide Behavior
+description: Configure the Menu's sub-items appearance by using the ShowOn parameter and control how they disappear through the HideOn option.
+slug: menu-show-hide-behavior
+tags: telerik,blazor,menu,show,hide,events,hover,click,mouseleave,mouseenter
+published: true
+position: 3
+---
+
+# Show and Hide Behavior
+
+You can control the user interaction with the Menu items by defining how the child items show and hide. 
+
+The `ShownOn` and `HideOn` parameters allow you to set the event that will show and hide the Menu items.
+
+* The `ShowOn` parameter accepts a value from the `Telerik.Blazor.MenuShowEvent` enum:
+
+    * `MouseEnter`&mdash;When the mouse cursor enters a Menu item, its sub-items will display.
+    * `Click`&mdash;When the user clicks or taps on a Menu item, its child items will display.
```

Comment:
It's good to use the same terms here, to avoid confusion (either sub items, or child items, but not both).